tests/core/test_views_transaction.py

```diff
@@ -71,6 +71,25 @@ def test_modifications_are_not_rolled_back_on_4XX_error(self):
         resp = self.app.get('/mushrooms', headers=self.headers)
         self.assertEqual(len(resp.json['data']), 1)
 
+    def test_modifications_are_rolled_back_on_409_error(self):
+        bucket_create = {
+            'method': 'POST',
+            'path': '/mushrooms',
+            'body': {'data': {'name': 'Vesse de loup', 'last_modified': 123}}
+        }
+        bucket_ccreate = {
+            'method': 'POST',
+            'path': '/mushrooms',
+            'body': {'data': {'name': 'Vesse de loup', 'last_modified': 123}}
+        }
+        body = {'requests': [bucket_create, bucket_ccreate]}
+        resp = self.app.post_json('/batch', body, headers=self.headers)
+        response = resp.json['responses'][1]
+        self.assertEqual(response['status'], 409)
+        resp = self.app.get('/buckets/mushrooms', headers=self.headers)
```

Comment:
Here we are in `kinto.core`, there is no notion of `buckets`. Try with `/mushrooms`